control-plane/grpc/src/lib.rs

```diff
@@ -18,3 +18,15 @@ pub(crate) mod pool {
 pub(crate) mod replica {
     tonic::include_proto!("v1.replica");
 }
+
+// Nexus GRPC module for the autogenerated pool code
+#[allow(clippy::enum_variant_names)]
```

Comment:
Could you doc why we needed to add this ?

Reply:
This is basically needed because Clippy wont allow same prefix and suffix to the enum members